lost/src/main/java/com/mapzen/android/lost/internal/LostClientManager.java

```diff
@@ -212,6 +212,10 @@ private void throwIfClientNotAdded(LostApiClient client) {
     return true;
   }
 
+  @Override public void shutdown() {
```

Comment:
If I'm recalling this change correctly, we landed on `shutdown` to make it a little more clear at the call site what this function's intended job is, with the intent to add future functionality. @sarahlensing might remember more.